Angular 12 Support

The current version of `ngx-vcard` supports Angular 11, please bring it up to support Angular 12

@DanielHabenicht the error given when attempting to install with angular 12 is
```
npm ERR!   @angular/common@"^12.1.1" from the root project
npm ERR!
npm ERR! Could not resolve dependency:
npm ERR! peer @angular/common@">=5.x <11.0.0" from ngx-vcard@3.0.22
```

--- a/projects/ngx-vcard/package.json
+++ b/projects/ngx-vcard/package.json
@@ -24,7 +24,7 @@
     "tslib": "2.3.1"
   },
   "peerDependencies": {
-    "@angular/common": ">=5.x <11.0.0",
-    "@angular/core": ">=5.x <11.0.0"
+    "@angular/common": ">=5.x <12.0.0",
+    "@angular/core": ">=5.x <12.0.0"
   }
 }

